domains/pallets/ethereum-beacon-client/src/ssz.rs

```diff
@@ -0,0 +1,155 @@
+use crate::config;
+use sp_std::{vec, vec::Vec};
+use ssz_rs::{
+	prelude::{List, Vector},
+	Bitlist, Bitvector, Deserialize, Sized, U256,
+};
+use ssz_rs_derive::SimpleSerialize;
+
+#[derive(Default, Debug, SimpleSerialize, Clone)]
+pub struct SSZVoluntaryExit {
```

Comment:
add more comments to these structures pls

Reply:
Added comments to the original generic data structures.